editor/styles/global.scss

```diff
@@ -44,9 +44,23 @@ $sidebarWidth: 4.6rem;
 [data-bs-theme="dark"] .app-navbar,
 .app-sidebar,
 .crash-injury-indicator {
-  background-color: $night-mode-background !important;
+  background-color: $dark-mode-background !important;
 }
 
+[data-bs-theme="dark"] .mapboxgl-popup-tip {
+  border-bottom-color: $dark-grey !important;
+  border-top-color: $dark-grey !important;
+}
```

Comment:
i did some quick styling updates here bc currently this is what the popups look like in prod if ur on dark mode :/

<img width="1696" height="1208" alt="image" src="https://github.com/user-attachments/assets/85309e80-5162-4dbd-ba6c-ed2e635843ca" />